source/partials/_nav.slim

```diff
@@ -5,20 +5,21 @@
   = nav_overlay_item('Apprenticeship', '/apprenticeship')
   = nav_overlay_item('Blog', '/blog')
 
-nav#main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}" aria-label=('Main navigation')
-  = link_to '/', class: 'Nav-logo'
-    .NavLogo class=nav_logo_class
+header
+  #main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}"
+    = link_to '/', "aria-label" => "Homepage", class: 'Nav-logo'
+      .NavLogo class=nav_logo_class
 
-  .Nav-menu
-    .Burger class=nav_burger_class
-      span.Burger-line
-      span.Burger-line
-      span.Burger-line
+    .Nav-menu
+      .Burger class=nav_burger_class
+        span.Burger-line
+        span.Burger-line
+        span.Burger-line
 
-  ul.Nav-items
-    = nav_item('Work', '/work')
-    = nav_item('Company', '/company')
-    = nav_item('Community', '/community')
-    = nav_item('Apprenticeship', '/apprenticeship')
-    = nav_item('Blog', '/blog')
-.u-navPlaceholder
+    nav aria-label='Main navigation' role='navigation'
```

Comment:
Not sure what you want me to do here. I don't share your concerns regarding having an element that is purely semantic, without a class, because I think those are two completely different scopes.

Moving the `Nav-items` up would mean having to change the CSS.